presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcConnectorFactory.java

```diff
@@ -58,7 +59,14 @@ public Connector create(String connectorId, Map<String, String> requiredConfig)
         requireNonNull(optionalConfig, "optionalConfig is null");
 
         try (ThreadContextClassLoader ignored = new ThreadContextClassLoader(classLoader)) {
-            Bootstrap app = new Bootstrap(new JdbcModule(connectorId), module);
+            Bootstrap app = new Bootstrap(
+                    new JdbcModule(connectorId),
+                    module,
+                    installModuleIf(
+                            JdbcSecurityConfig.class,
+                            security -> "none".equalsIgnoreCase(security.getPostgreSqlSecuritySystem()),
```

Comment:
If Presto is connected to a Mysql database and a Postgres database, what if we want to have different security configurations for each one? Would this situation not arise?